packages/config/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import traverse from 'traverse';
+import {Module, ConstantProvider} from '@sensejs/core';
+
+export interface ConfigModuleOption {
+  config: object;
+  prefix: string;
+}
+
+function escape(x: string) {
+  return x.replace('\\', '\\\\').replace('.', '.');
```

Comment:
`.replace('.', '.')`
carelessness or on purpose

and this map function should have a test?